tests/integration/test_storage_delta/test.py

```diff
@@ -488,9 +607,14 @@ def test_restart_broken(started_cluster, use_delta_kernel):
 
     write_delta_from_file(spark, parquet_data_path, f"/{TABLE_NAME}")
     upload_directory(minio_client, bucket, f"/{TABLE_NAME}", "")
-    create_delta_table(
-        instance, TABLE_NAME, bucket=bucket, use_delta_kernel=use_delta_kernel
-    )
+
+    instance.query(
+        f"""
+        DROP TABLE IF EXISTS {TABLE_NAME};
+        CREATE TABLE {TABLE_NAME}
+        ENGINE=DeltaLake(s3, filename = '{TABLE_NAME}/', url = 'http://minio1:9001/{bucket}/')
+        SETTINGS allow_experimental_delta_kernel_rs={use_delta_kernel}""")
```

Comment:
As I see `create_delta_table` now supports arbitrary `bucket` passed via args, so let's use it?

Reply:
oh right, ah missed it, thanks